cmd/notation/cache.go

```diff
@@ -0,0 +1,235 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/notaryproject/notation/pkg/config"
+	"github.com/notaryproject/notation/pkg/registry"
+	"github.com/opencontainers/go-digest"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	cacheCommand = &cli.Command{
+		Name:  "cache",
+		Usage: "Manage signature cache",
+		Subcommands: []*cli.Command{
+			cacheListCommand,
+			cachePruneCommand,
+			cacheRemoveCommand,
+		},
+	}
+
+	cacheListCommand = &cli.Command{
+		Name:    "list",
+		Usage:   "List signatures in cache",
+		Aliases: []string{"ls"},
+		Flags: []cli.Flag{
+			localFlag,
+			usernameFlag,
+			passwordFlag,
+			plainHTTPFlag,
+		},
+		ArgsUsage: "[reference|manifest_digest]",
+		Action:    listCachedSignatures,
+	}
+
+	cachePruneCommand = &cli.Command{
+		Name:      "prune",
+		Usage:     "Prune signature from cache",
+		ArgsUsage: "[reference|manifest_digest] ...",
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:    "all",
+				Aliases: []string{"a"},
+				Usage:   "prune all cached signatures",
+			},
+			&cli.BoolFlag{
+				Name:    "force",
+				Aliases: []string{"f"},
+				Usage:   "prune entire directory, combined with --all",
```

Comment:
force is usually used to disable the "prompt for confirmation"

Reply:
Now, it is naned `--purge`. `force` is for confirmation.